recipes/cmdstan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "cmdstan" %}
+{% set version = "2.27.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stan-dev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ff71f4d255cf26c2d8366a8173402656a659399468871305056aa3d56329c1d5
+
+build:
+  number: 0
+  binary_relocation: False
+
+requirements:
+  build:  
+    - {{ compiler('cxx') }}  # [not win]
+    - make                   # [not win]
+    - tbb-devel =2020.2      # [not win]
+    - m2w64-toolchain        # [win]
+    - m2-filesystem          # [win]
+    - m2-sed                 # [win]
+    - m2-coreutils           # [win]
+  host:
+    - tbb-devel =2020.2      # [not win]
+  run:
+    - {{ compiler('cxx') }}  # [not win]
+    - make                   # [not win]
```

Comment:
Why do you need this in `run`?

Reply:
CmdStan is sort of an odd duck - it is a tool for compiling models in the stan language, so it's one part library and one part build script. The actual usage of the tool at runtime requires Make and a c++ toolchain. The [docs](https://mc-stan.org/docs/2_24/cmdstan-guide/compiling-a-stan-program.html) go into more detail, if you're curious, but essentially a `.stan` program is translated into a cpp file, which is then linked with the stan library and compiled to be run. CmdStan handles the first half of this, and provides a build script for the last portion, but it assumes you have a compiler to do so